tests/jcond.s

```diff
@@ -0,0 +1,31 @@
+jmp #5                      ; keep pass/fail at fixed location
+print ^cond_jump_insts_ok   ; overall pass
+halt
+print ^cond_jump_insts_fail ; any fail
+halt
+sub $1 $2 #3                ; test all j insts that don't consider < case as match
```

Comment:
The jump instruction you propose it to let operand 1 compare with 0, but your test logic is to compare with some calculate result. It can be reduced to simply test each operator with -1, 0, 1 like this:

```
sub $0 $1 #1  ; prepare -1
add $0 $0 #2  ; prepare 0
add $0 $1 #3  ; prepare 1
jz #1 #3           ; jz -1 == 0
jz #2 #5           ; jz  0 == 0, jump to next instruction
jz #3 #3           ; jz 1 == 1
```